src/storage/btree.c

```diff
@@ -1792,6 +1792,76 @@ static DISK_ISVALID btree_check_tree (THREAD_ENTRY * thread_p, const OID * class
 static DISK_ISVALID btree_check_by_btid (THREAD_ENTRY * thread_p, BTID * btid);
 static char *btree_unpack_mvccinfo (char *ptr, BTREE_MVCC_INFO * mvcc_info, short btree_mvcc_flags);
 
+static int btree_is_key_visible (THREAD_ENTRY * thread_p, BTID_INT * btid, PAGE_PTR pg_ptr,
+				 MVCC_SNAPSHOT * mvcc_snapshot, int slot_id, bool * is_visible, DB_VALUE * key_value);
+
+/*
+ *  btree_is_key_visible(): States if current key is visible or not.
+ *
+ *  thread_p(in): Thread entry.
+ *  btid(in): B-tree info.
+ *  pg_ptr(in):	Page pointer.
+ *  mvcc_snapshot(in): The MVCC snapshot.
+ *  slot_id(in) : Slot id to be looked for.
+ *  is_visible(out): True or False
+ *
+ *  return: error code if any error occurs.
+ */
+static int
+btree_is_key_visible (THREAD_ENTRY * thread_p, BTID_INT * btid, PAGE_PTR pg_ptr, MVCC_SNAPSHOT * mvcc_snapshot,
```

Comment:
The clear unit test case with Isolation Level is needed to validate this function.

Reply:
There is posted a test case which is written by cci-driver on CBRD-24436 as like count(*). The code hase 3 tranaction group having 50 thread running , which are DML (insert/delete), MIN (select min), MAX (select MAX).